src/app/main.cpp

```diff
@@ -148,6 +148,7 @@ void usage( const QString &appName )
       << QStringLiteral( "\t[--nologo]\thide splash screen\n" )
       << QStringLiteral( "\t[--noversioncheck]\tdon't check for new version of QGIS at startup\n" )
       << QStringLiteral( "\t[--noplugins]\tdon't restore plugins on startup\n" )
+      << QStringLiteral( "\t[--skipbadlayers]\tdon't prompt for missing layers\n" )
```

Comment:
Once that's merged, we should prepare another PR to list the shorthand commands here as well, but that's better a coordinated effort:

```
  ...
  -B, --skipbadlayers    don't prompt for missing layers
  ...
```